# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/base/nsBaseChannel.h
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c netwerk/base/nsBaseChannel.h
--- a/netwerk/base/nsBaseChannel.h	Tue Feb 12 17:58:19 2019 +0100
+++ b/netwerk/base/nsBaseChannel.h	Tue Feb 12 16:08:25 2019 +0000
@@ -94,7 +94,7 @@
 
   // Implemented by subclass to begin pumping data for an async channel, in
   // lieu of returning a stream. If implemented, OpenContentStream will never
-  // be called for async channels. If not implemented, AsyncOpen2 will fall
+  // be called for async channels. If not implemented, AsyncOpen will fall
   // back to OpenContentStream.
   //
   // On success, the callee must begin pumping data to the stream listener,
@@ -222,7 +222,6 @@
   // pointers.
   void ChannelDone() {
     mListener = nullptr;
-    mListenerContext = nullptr;
     OnChannelDone();
   }
 
@@ -279,7 +278,6 @@
   nsCOMPtr<nsILoadInfo> mLoadInfo;
   nsCOMPtr<nsIInterfaceRequestor> mCallbacks;
   nsCOMPtr<nsIStreamListener> mListener;
-  nsCOMPtr<nsISupports> mListenerContext;
   nsresult mStatus;
   uint32_t mContentDispositionHint;
   nsAutoPtr<nsString> mContentDispositionFilename;